vertx-mysql-client/src/main/java/io/vertx/mysqlclient/MySQLResult.java

```diff
@@ -0,0 +1,15 @@
+package io.vertx.mysqlclient;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.sqlclient.SqlClient;
+import io.vertx.sqlclient.SqlResultProperty;
+
+/**
+ *
+ */
+@VertxGen
+public interface MySQLResult {
+  SqlResultProperty<Integer> LAST_INSERTED_ID = new SqlResultProperty<Integer>() {
```

Comment:
rather on `MySQLClient`, `MySQLResult` is confusing and make think that this class should be used as result specifically for MySQL and it is not